test/unit-test/cellular_pktio_utest.c

```diff
@@ -1650,7 +1647,7 @@ void test__Cellular_PktioSendAtCmd_Null_AtCmd( void )
     CellularPktStatus_t pktStatus = CELLULAR_PKT_STATUS_OK;
     CellularContext_t context;
     CellularCommInterface_t * pCommIntf = &CellularCommInterface;
-    CellularCommInterfaceHandle_t commInterfaceHandle;
+    CellularCommInterfaceHandle_t commInterfaceHandle = ( CellularCommInterfaceHandle_t ) malloc( sizeof( CellularCommInterfaceHandle_t ) );
```

Comment:
what happens when this malloc fails?

Reply:
That will cause error message after setting compile option -Werror
```
/mnt/c/Andy/project/cellular/lib/Lab-Project-FreeRTOS-Cellular-Library/test/unit-test/cellular_pktio_utest.c: In function ‘test__Cellular_PktioSendAtCmd_Null_AtCmd’:
/mnt/c/Andy/project/cellular/lib/Lab-Project-FreeRTOS-Cellular-Library/test/unit-test/cellular_pktio_utest.c:1664:28: error: ‘commInterfaceHandle’ is used uninitialized in this function [-Werror=uninitialized]
 1664 |     context.hPktioCommIntf = commInterfaceHandle;
      |     ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[2]: *** [unit-test/CMakeFiles/cellular_pktio_utest.dir/build.make:67: unit-test/CMakeFiles/cellular_pktio_utest.dir/cellular_pktio_utest.c.o] Error 1
```

Reply:
please refer to this [change](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/pull/37/commits/d6ed9baaca207edc8cb5ab7fe890e2937975d2d8).